src/System.Memory/tests/ReadOnlySpan/IndexOf.T.cs

```diff
@@ -193,5 +193,13 @@ public static void IndexOfNull_String(string[] spanInput, int expected)
             ReadOnlySpan<string> theStrings = spanInput;
             Assert.Equal(expected, theStrings.IndexOf((string)null));
         }
+
+        [Theory]
+        [MemberData(nameof(TestHelpers.ContainsNullData), MemberType = typeof(TestHelpers))]
+        public static void ContainsNull_String(string[] spanInput, bool expected)
+        {
+            ReadOnlySpan<string> theStrings = spanInput;
+            Assert.Equal(expected, theStrings.Contains(null));
+        }
```

Comment:
Where are the rest of the Contains tests?  This file only contains IndexOf tests.

Reply:
We don't have contain tests for Span<T>. I will add that. but we Contains.T file that contains test for ReadOnlySpan. I will move that test to that file.